README.md

```diff
@@ -18,21 +18,39 @@ with all required dependencies and then run the script to preprocess the project
 Simply commit your changes and do a git push.  There is also `build/publish.sh` which integrates with the rook `Makefile`
 nicely, and is meant to be run in conjunction with Jenkins CI -- and expects environment variable `DOCS_VERSION`.
 
-# Run
+# Test
 
-To test your changes to the Rook website locally before committing, you can run the `build/run.sh` script. The script
-requires Docker installed on your machine.
-When running the script, it will start a Jekyll Docker container which will then build and rebuild the docs on changes
-to the `_site/` directory. Viewing the docs in your browser can be achieved by running:
+To test your changes to the Rook website locally before committing to the Rook repo, you can generate the docs locally and start a local web server. 
+
+To generate the docs:
+```bash
+# Remove the old master docs and copy your updated docs from your rook repo
+rm -fr docs/rook/master
+mkdir docs/rook/master
+cp -r ../rook/Documentation/* docs/rook/master/
```

Comment:
Could you change it to symlinking `ln -s` that would help directly developing the docs and not having to copy changes back and forth.